overlay/scikitlearn_validation/validation.py

```diff
@@ -29,12 +31,48 @@ def __init__(self, request: ValidationJobRequest):
         self.sample_rate = request.sample_rate
         info(f"ScikitLearnValidator(): limit={self.limit}, sample_rate={self.sample_rate}")
 
-    def load_sklearn_model(self, verbose=False):
+    def load_sklearn_model(self, verbose=True):
         # Load ScikitLearn model from disk
         model_path = f"{MODELS_DIR}/{self.job_id}"
-        info(f"Loading ScikitLearn model from {model_path}")
+
+        # pick the first file. only one file (pickled object) is supposed to be in the directory.
+        file_name = os.listdir(model_path)[0]
+
+        model_path += f'/{file_name}'
+
         model = pickle.load(open(model_path, 'rb'))
-        # TODO: use verbose
+
+        info(f"Loading ScikitLearn model from {model_path}")
+
+        if verbose:
+            model_type = type(model).__name__
+            info(f"Model type (from binary): {model_type}")
+            if model_type == "LinearRegression":
+                info(f"Model Description: Coefficients: {model.coef_}, Intercept: {model.intercept_}")
+            elif model_type == "GradientBoostingRegressor":
+                info(f"Model Description(feature_importances: {model.feature_importances_},"
+                     f"oob_improvement: {model.oob_improvement_},"
+                     f"train_score: {model.train_score_},"
+                     f"loss: {model.losee_},"
+                     f"init_: {model.init_},"
+                     f"estimators: {model.estimators_},"
+                     f"n_classes: {model.n_clases_},"
+                     f"n_estimators: {model.n_estimators_},"
+                     f"n_features: {model.n_features_},"
+                     f"max_features: {model.max_features_})")
+            elif model_type == "SVR":
```

Comment:
can we make this type `SupportVectorRegressor` to avoid acronyms?

Reply:
This is the class name used in SKLearn's internal libraries (returned by `type(model).__name__`)